media/js/custom.js

```diff
@@ -93,3 +93,19 @@ $(function() {
         $('.academics-' + thisItem).show();
     });
 });
+
+$(function () {
+	$('.change-year-select').change(function () {
+		var currentYear = $(this).val();
+		$('#honours-board-tab li').removeClass('active');
+		$('.default-tab').addClass('active');
+		$('.tab-pane').removeClass('active');
+		$('#staff'+currentYear).addClass('active');
+		$('.tab-content').hide();
+		$('.board-'+currentYear).show();
+		$('[data-toggle = "tab"]').each(function (i, elem) {
```

Comment:
We don't use element indices here. So why bother creating the `i` variable at all? Use `_` in place of `i`.